javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -344,7 +348,7 @@ class LinkedList[E]() extends AbstractSequentialList[E]
 
   def descendingIterator(): Iterator[E] = {
     new Iterator[E] {
-      private val iter = listIterator(_size)
+      private val iter = listIterator(_size.toInt)
```

Comment:
This would not work if the list has effectively more than `Int.MaxValue` elements.